simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java

```diff
@@ -20,15 +28,29 @@
 public class DropwizardExports extends io.prometheus.client.Collector implements io.prometheus.client.Collector.Describable {
     private static final Logger LOGGER = Logger.getLogger(DropwizardExports.class.getName());
     private MetricRegistry registry;
+    private MetricFilter metricFilter;
     private SampleBuilder sampleBuilder;
 
     /**
-     * Creates a new DropwizardExports with a {@link DefaultSampleBuilder}.
+     * Creates a new DropwizardExports with a {@link DefaultSampleBuilder} and {@link MetricFilter#ALL}
```

Comment:
Fullstop

Reply:
What do you mean by "Fullstop"? Could you please clarify a bit?

Reply:
Ahh, sorry, I'm not a native speaker, didn't understand word "fullstop".